### PR TITLE
Update page in APIM release note section “20201130_apip_relnotes”

### PR DESCRIPTION
**Pull request by jowens-axway**
https://deploy-preview-1441--axway-open-docs.netlify.app/docs/apim_relnotes/20201130_apip_relnotes/

### CODE REVIEW

File: content/en/docs/apim_relnotes/20201130_apip_relnotes.md [hunk 1]
```diff
@@ -18,9 +18,44 @@ API Portal is available as a software installation or a virtualized deployment i
 
 ## New features and enhancements
 
-### Easy upgrade from 7.5.5 and 7.6.2 to 7.7 November
+### Production Ready Docker Container - Phase 1
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20201130_apip_relnotes.md:21]: @jowens-axway @dginevaAxway is there a page in the docs, which I could link from this section?

File: content/en/docs/apim_relnotes/20201130_apip_relnotes.md [hunk 2]
```diff
@@ -18,9 +18,44 @@ API Portal is available as a software installation or a virtualized deployment i
...
+* Virus scanning of uploaded files via the Portal interface.
+* In all log files generated by API Portal now, personally identifiable information (PII) is now protected using Global Unique Identifiers (GUIDs). Each user in API Portal is associated to a GUID, which is used instead of the user name to protect the user data in compliance with General Data Protection Regulation (GDPR).
+
+### Cumulative Upgrade Script
+
+* In July 2020, we released a "cumulative" upgrade script, `apiportal_cumulative_upgrade.sh`to help customers transition from 7.5.5 / 7.6 up to 7.7 July 2020​ update.
+* This script has been updated to upgrade customers to the 7.7 November 2020 update inline with the end of support for the 7.5.x and 7.6.x product lines​ in November 2020.
+* For more information, see [Upgrade API Portal using the cumulative upgrade script](/docs/apim_installation/apiportal_install/upgrade_automatic/#upgrade-api-portal-using-the-cumulative-upgrade-script).
+
+### User Experience Improvements
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20201130_apip_relnotes.md:51]: @jowens-axway @dginevaAxway is there a page in the docs, which I could link from this section?

File: content/en/docs/apim_relnotes/20201130_apip_relnotes.md [hunk 3]
```diff
@@ -18,9 +18,44 @@ API Portal is available as a software installation or a virtualized deployment i
 
 ## New features and enhancements
 
-### Easy upgrade from 7.5.5 and 7.6.2 to 7.7 November
+### Production Ready Docker Container - Phase 1
 
-With API Portal 7.5.5 and 7.6.2 reaching end of support in November 2020, we are looking at ways to help customers make the jump to the latest 7.7 update.​ The `apiportal_cumulative_upgrade.sh` is a cumulative script that you can use to upgrade an environment from 7.5.5 or 7.6.2 directly to the latest 7.7 update​. For more information, see [Upgrade API Portal using the cumulative upgrade script](/docs/apim_installation/apiportal_install/upgrade_automatic/#upgrade-api-portal-using-the-cumulative-upgrade-script).
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20201130_apip_relnotes.md:23]: @jowens-axway did you actually removed this information?

File: content/en/docs/apim_relnotes/20201130_apip_relnotes.md (hunk 2 above)
Comment (jowens-axway) [reply content/en/docs/apim_relnotes/20201130_apip_relnotes.md:51]: its already linked to a page in the docs, see the link under "Upgrade API Portal using the cumulative upgrade script"

File: content/en/docs/apim_relnotes/20201130_apip_relnotes.md (hunk 3 above)
Comment (jowens-axway) [reply content/en/docs/apim_relnotes/20201130_apip_relnotes.md:23]: I rewrote some of this paragraph to make it more applicable to the actual change in the Nov update, this looks like it was copy and pasted from a previous release notes.

Comment (andreamussap) [reply content/en/docs/apim_relnotes/20201130_apip_relnotes.md:23]: No problem. I just wanted to make sure. @dginevaAxway FYI.

**Comment by andreamussap** [comment]
@jowens-axway I've made changes for Axway style and improve readability. Can you please check that you're ok with them, so I can merge the PR? Thanks.

**Comment by jowens-axway** [comment]
> @jowens-axway I've made changes for Axway style and improve readability. Can you please check that you're ok with them, so I can merge the PR? Thanks.

have you got a link where i can see what it looks like? sorry, cant find the correct link anywhere in here!

**Review by andreamussap — APPROVED** [review]

**Pull request merged**